### PR TITLE
Add config flow to linky

### PR DESCRIPTION
**Pull request by Quentame**
**Description:**

Documentation for : https://github.com/home-assistant/home-assistant/pull/26076

**Pull request in home-assistant :** https://github.com/home-assistant/home-assistant/pull/26076

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10284"></a>

### CODE REVIEW

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_components/linky.markdown [hunk 1]
```diff
@@ -13,17 +13,24 @@ redirect_from:
...
+
+Add the Linky sensor to your `configuration.yaml` file like below:
 
 ```yaml
 # Example configuration.yaml entry
-sensor:
-  - platform: linky
-    username: YOUR_LINKY_USERNAME
+linky:
+  - username: YOUR_LINKY_USERNAME
```
Comment (klaasnicolaas) [inline source/_components/linky.markdown:33]:
```suggestion
  - username: YOUR_LINKY_USERNAME
```

or

```suggestion
  username: YOUR_LINKY_USERNAME
```
?

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/linky.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/linky.markdown:33]: @klaas, it expects a list so the current documentation is correct.

**Review by frenck — APPROVED** [review]
Thanks, @Quentame! 👍

File: source/_components/linky.markdown (hunk 1 above)
Comment (Quentame) [reply source/_components/linky.markdown:33]: Exactly 😉

**Pull request merged**